Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -59,14 +59,20 @@
         //Sandcastle_Begin
```

Comment:
I see a slight difference between the environment map in this branch and master. Is the image rotated differently? Normally this wouldn't really matter but the spherical harmonics are no longer aligned with the environment map.

![ibl_difference](https://user-images.githubusercontent.com/915398/122145710-c0dfc600-ce23-11eb-8b53-8f0e9f64183e.jpg)

Reply:
not sure what a good workaround here is.. the Khronos tool does not generate spherical harmonics.

Reply:
doesn't look like Filament's spherical harmonics match the Khronos sampler's output - 
![image](https://user-images.githubusercontent.com/31491650/122988364-a9e32b80-d36f-11eb-9a1f-8eb6c04029f3.png)

there are also weird patterns at certain angles both without spherical harmonics and with the misaligned ones - 
![image](https://user-images.githubusercontent.com/31491650/122989024-58876c00-d370-11eb-9d14-4b1ffa98cb0d.png)

Reply:
The spherical harmonic problem was fixed in https://github.com/CesiumGS/cesium/pull/9513/commits/e6b833d6be9e5fc12df97614a86063e060b788dd by updating the shader code to match to the newer filament code.